clang/test/Parser/cxx-concepts-requires-clause.cpp

```diff
@@ -168,3 +168,30 @@ auto lambda4 = [] requires(sizeof(char) == 1){}; // expected-error {{expected bo
 #if __cplusplus <= 202002L
 // expected-warning@-2{{lambda without a parameter clause is a C++23 extension}}
 #endif
+
+namespace GH78524 {
+
+template <typename T> T Foo;
+
+template <typename T> auto C(Foo<T>);
+
+template <typename T> struct D {
+  decltype(T()(C<T>)) Type;
+};
+
+template <typename T, typename U> D<T> G(T, U) { return {}; }
+
+struct E {};
+
+void F() {
+  G([]<typename T>
+//     ~~~~~~~~~~ T: 0,0
```

Comment:
In these context of this PR these comments about depth make sense but for someone coming to this test outside of this context they won't. Can you add more "depth" to the comment :-)

Reply:
Oh, I was going to detail the comment after putting up the PR, but for some reason I forgot it. Added.